apps/ehr/src/components/AppointmentTabs.tsx

```diff
@@ -97,106 +97,104 @@ export default function AppointmentTabs({
-                inHouseLabOrdersByAppointmentId={inHouseLabOrdersByAppointmentId}
-                externalLabOrdersByAppointmentId={externalLabOrdersByAppointmentId}
-                nursingOrdersByAppointmentId={nursingLabOrdersByAppointmentId}
-                location={location}
-                tab={value}
-                now={now}
-                updateAppointments={updateAppointments}
-                setEditingComment={setEditingComment}
-              ></AppointmentTable>
-            )}
-          </TabPanel>
-          <TabPanel value={ApptTab.cancelled} sx={{ padding: 0 }}>
-            {selectLocationMsg || (
-              <AppointmentTable
-                appointments={cancelledAppointments}
-                inHouseLabOrdersByAppointmentId={inHouseLabOrdersByAppointmentId}
-                externalLabOrdersByAppointmentId={externalLabOrdersByAppointmentId}
-                nursingOrdersByAppointmentId={nursingLabOrdersByAppointmentId}
-                location={location}
-                tab={value}
-                now={now}
-                updateAppointments={updateAppointments}
-                setEditingComment={setEditingComment}
-              ></AppointmentTable>
-            )}
-          </TabPanel>
-        </TabContext>
-      </Box>
-    </>
+    <Box sx={{ width: '100%', marginTop: 3 }}>
+      <TabContext value={value}>
+        <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
+          <TabList
+            variant="scrollable"
+            allowScrollButtonsMobile={true}
+            onChange={handleChange}
+            aria-label="appointment tabs"
+          >
+            <Tab
+              data-testid={dataTestIds.dashboard.prebookedTab}
+              label={`Pre-booked${preBookedAppointments ? ` – ${preBookedAppointments?.length}` : ''}`}
+              value={ApptTab.prebooked}
+              sx={{ textTransform: 'none', fontWeight: 500 }}
+            />
+            <Tab
+              data-testid={dataTestIds.dashboard.inOfficeTab}
+              label={`In Office${inOfficeAppointments ? ` – ${inOfficeAppointments?.length}` : ''}`}
+              value={ApptTab['in-office']}
+              sx={{ textTransform: 'none', fontWeight: 500 }}
+            />
+            <Tab
+              data-testid={dataTestIds.dashboard.dischargedTab}
+              label={`Discharged${completedAppointments ? ` – ${completedAppointments?.length}` : ''}`}
+              value={ApptTab.completed}
+              sx={{ textTransform: 'none', fontWeight: 500 }}
+            />
+            <Tab
+              data-testid={dataTestIds.dashboard.cancelledTab}
+              label="Cancelled"
+              value={ApptTab.cancelled}
+              sx={{ textTransform: 'none', fontWeight: 500 }}
+            />
+            {loading && <Loading />}
+          </TabList>
+        </Box>
+        <TabPanel value={ApptTab.prebooked} sx={{ padding: 0 }}>
+          {selectLocationMsg || (
+            <AppointmentTable
```

Comment:
i know this isn't part of your changes, but these 4 could be made into a helper function that takes in an appointments array with the rest of the items in the closure of the component